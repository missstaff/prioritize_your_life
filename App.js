import { StatusBar } from "expo-status-bar";
import { SafeAreaProvider } from "react-native-safe-area-context";
import MasterFlow from "./src/components/navigation/MasterFlow";
import { useColorScheme } from "react-native";


const App = () => {
  const scheme = useColorScheme();
  const statusBarStyle = scheme === "dark" ? "light" : "dark";
  return (
    <SafeAreaProvider>
      <MasterFlow />
      <StatusBar style={statusBarStyle} />
    </SafeAreaProvider>
  );
};

export default App;
