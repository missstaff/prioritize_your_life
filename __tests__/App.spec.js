import renderer from "react-test-renderer";
import App from "../App";


describe("App", () => {
  afterAll(() => {
    jest.clearAllMocks();
  });
  it("App should render correctly", () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});