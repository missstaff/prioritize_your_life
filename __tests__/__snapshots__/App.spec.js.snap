// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
